### PR TITLE
PigPinBoyz

### PR DESCRIPTION
**Pull request by jbiggznyc**
Where  men come to party & play

### CODE REVIEW

**Comment by jbiggznyc** [comment]
Thanks

File: .github/workflows/google.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    - name: Build
+      run: |        
+        docker build -t "$REGISTRY_HOSTNAME"/"$GKE_PROJECT"/"$IMAGE":"$GITHUB_SHA" \
+          --build-arg GITHUB_SHA="$GITHUB_SHA" \
+          --build-arg GITHUB_REF="$GITHUB_REF" .
+
+    # Push the Docker image to Google Container Registry
+    - name: Publish
+      run: |
+        docker push $REGISTRY_HOSTNAME/$GKE_PROJECT/$IMAGE:$GITHUB_SHA
```
Comment (jbiggznyc) [inline .github/workflows/google.yml:60]: Looks fairly normal

**Review by nomisreid — COMMENTED** [review]
IMG_20211115_193457

File: .github/workflows/google.yml
Comment (zfnmdfgh62-cmd) [inline .github/workflows/google.yml:1]: How can I join?

**Pull request closed without merging**